x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/unsaved_timeline.cy.ts

```diff
@@ -38,10 +38,12 @@ import {
 } from '../../../tasks/timeline';
 import { EXPLORE_URL, hostsUrl, MANAGE_URL } from '../../../urls/navigation';
 
-// FLAKY: https://github.com/elastic/kibana/issues/169588
-describe.skip('Save Timeline Prompts', { tags: ['@ess'] }, () => {
-  before(() => {
+describe('Save Timeline Prompts', { tags: ['@ess'] }, () => {
+  beforeEach(() => {
     login();
+    visitWithTimeRange(hostsUrl('allHosts'));
```

Comment:
might not be necessary, but wouldn't we want to delete all created timelines before doing anything, using this [deleteTimelines](https://github.com/elastic/kibana/blob/main/x-pack/test/security_solution_cypress/cypress/tasks/api_calls/common.ts#L116) function?

Reply:
I don't think that is necessary since it doesn't matter to the test if there are other timelines.